src/interfaces/stepper-interfaces/PeopleAndRoles/org-person-interface.ts

```diff
@@ -1,31 +1,22 @@
 import { AddressIF, RoleIF } from '@/interfaces'
-import { ActionTypes, IncorporatorTypes, RoleTypes } from '@/enums'
+import { ActionTypes, PartyTypes, RoleTypes } from '@/enums'
 
 export interface OrgPersonIF {
   officer: {
     id: string
-    partyType: IncorporatorTypes
+    partyType: PartyTypes
     firstName: string
     middleName?: string
     lastName: string
-    organizationName: string
+    organizationName?: string
     email?: string
   }
```

Comment:
Do you need `businessNumber` or `incorpNumber` here as well for prorietor-orgs ?

Reply:
Probably yes, will be done the following commit focused on the Corp Partners